network_wrangler/models/gtfs/tables.py

```diff
@@ -102,9 +102,9 @@ class StopsTable(pa.DataFrameModel):
     - uniqueness: ["stop_id"]
     """
 
-    stop_id: Series[str] = pa.Field(coerce=True, nullable=False, unique=True)
+    stop_id: Series[str] = pa.Field(coerce=True, nullable=False, unique=False)
     # TODO why is that here?
-    model_node_id: Series[int] = pa.Field(coerce=True, nullable=False)
```

Comment:
rail nodes are missing model_node_id. Adding rail links and nodes to standard network can solve this issue